### PR TITLE
docs: document the process for tracking security sensitive issues

### PR DESCRIPTION
**Pull request by akhi3030**
https://docs.google.com/document/d/14w1Sdk1wXBTJIuzEksl1U1X_tMtvjn3fwdaeP_mIdPE/edit is the internal document being exported here.

### CODE REVIEW

File: docs/practices/security_sensitive_issues.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   this repository is restricted to the set of people who are trusted to work on
+   and have knowledge about security sensitive issues pertaining to nearcore.
+
+   This repository can manually synced with the public nearcore repository using
+   the following commands:
+
+       git remote add nearcore-public git@github.com:near/nearcore
+       git remote add nearcore-private git@github.com:near/nearcore-private
+       git fetch nearcore-public
+       git push nearcore-private nearcore-public/master:master
```
Comment (matklad) [inline docs/practices/security_sensitive_issues.md:21]:
````suggestion
    ```  console
    $ git remote add nearcore-public git@github.com:near/nearcore
    $ git remote add nearcore-private git@github.com:near/nearcore-private
    $ git fetch nearcore-public
    $ git push nearcore-private nearcore-public/master:master
    ```
````

style issues: 

* stick to fenced code blocks
* use `console` as a language and a leading `$` to emphasize that this is an interactive shell command.

**Review by matklad — APPROVED** [review]

File: docs/practices/security_sensitive_issues.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+2. All security sensitive issues must be created on this repository.  You must
+   also assign one of the `[P-S0, P-S1]` labels to the issue to indicate the severity
+   of the issue.  The two criteria to use to help you judge the severity are
+   ease of carrying out the attack and the impact of the attack.  An attack that
+   is easy to do or can have a huge impact should have the `P-S0` label and `P-S1`
+   otherwise.
+
+3. All security sensitive pull requests should also be created on this
+   repository.  Note that once a PR is approved on the private repository, it
+   should not be merged into the private repository.  Instead it should be first
```
Comment (nagisa) [inline docs/practices/security_sensitive_issues.md:32]:
```suggestion
3. All security sensitive pull requests should also be created on the private
   repository.  Note that once a PR has been approved, it
   should not be merged into the private repository.  Instead it should be first
```

**Review by nagisa — APPROVED** [review]

File: docs/practices/security_sensitive_issues.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+## Security sensitive issues
```
[the remark this answers is not in the document]
Comment (akhi3030) [reply docs/practices/security_sensitive_issues.md]: We are actually planning on moving away from security@near.org.  But that will take some time.  I suppose I can document that for now and have to remember to update it when it changes.

File: docs/practices/security_vulnerabilities.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+## Security Vulnerabilities
+
+<blockquote style="background: rgba(255, 200, 0, 0.1); border: 5px solid rgba(255, 200, 0, 0.4);">
+The intended audience of the information presented here are developers working
```
Comment (nagisa) [inline docs/practices/security_vulnerabilities.md:4]: Worth checking if this needs a whitespace between tags and contents. IIRC markdown needs a space to generate a `<p>` for the text and/or to interpret html-looking stuff as html, but that may just be github-flavoured markdown.

Comment (nagisa) [reply docs/practices/security_vulnerabilities.md:4]: But we can also just fix that afterwards if its broken too.

Comment (akhi3030) [reply docs/practices/security_vulnerabilities.md:4]: Good idea.  It appeared fine in my local setup.  I'll double check once it is merged.

Comment (akhi3030) [reply docs/practices/security_vulnerabilities.md:4]: The render on current master is at https://near.github.io/nearcore/practices/security_vulnerabilities.html.  It looks fine to me, so I suppose we are fine here.

**Pull request merged**